main/server/models.py

```diff
@@ -1,40 +1,46 @@
 from main.server import app, db, ma
 from marshmallow import fields
 
+
 class Gallery(db.Model):
     __tablename__ = 'GALLERY'
-    artworkID = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    artworkLink = db.Column(db.String(2048), nullable=False)
-    artistLink = db.Column(db.String(2048), nullable=True)
+    artworkID = db.Column(db.Integer, 
+                primary_key=True, 
+                autoincrement=True)
+    artworkLink = db.Column(db.String(64), nullable=False)
+    blurhash = db.Column(db.String(64), nullable=True)
+    artistLink = db.Column(db.String(64), nullable=True)
     username = db.Column(db.String(64), nullable=True)
-    title = db.Column(db.String(64), nullable=True)
-    __table_args__ = (db.UniqueConstraint('artworkLink'),
-            )
+    # 256 because title can be a message too
+    title = db.Column(db.String(256), nullable=True)
 
-    def __init__(self, artworkLink, username, title, artistLink):
+    def __init__(self, artworkLink, blurhash, username, title, artistLink):
         self.artworkLink = artworkLink
+        self.blurhash = blurhash
         self.artistLink = artistLink
         self.username = username
         self.title = title
 
+
 class GallerySchema(ma.Schema):
     artworkID = fields.Integer()
     artworkLink = fields.String(required=True)
+    blurhash = fields.String(required=True)
     artistLink = fields.String(required=False)
     username = fields.String(required=True)
     title = fields.String(required=False)
 
 
 class Games(db.Model):
     __tablename__ = 'GAMES'
-    gameID = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    gameLink = db.Column(db.String(2048), nullable=False)
-    gitLink = db.Column(db.String(2048), nullable=True)
+    gameID = db.Column(db.Integer, 
+                primary_key=True, 
+                autoincrement=True)
+    gameLink = db.Column(db.String(64), nullable=False)
+    gitLink = db.Column(db.String(64), nullable=True)
```

Comment:
We might have very long urls (e.g. google doc urls are ~120 characters). Might be safer to go with something a bit higher.

Reply:
256 should be good, probably